CHANGELOG.md

```diff
@@ -14,6 +14,15 @@
 
 🆕 New features:
 
+- Add new govuk-shade and govuk-tint functions for creating shades and tints of
+  colours.
+
+  ([PR #1207](https://github.com/alphagov/govuk-frontend/pull/1207))
+
+- Add new secondary and warning button variants
+
+  ([PR #1207](https://github.com/alphagov/govuk-frontend/pull/1207))
```

Comment:
Do we need to separate entries?

Reply:
Whilst one relies on the other, I think they're two distinct things that should be called out separately?